iphone/Classes/DeMarcelpociotCollectionviewCollectionView.h

```diff
@@ -8,7 +8,6 @@
 #import "TiUIView.h"
 #import "DeMarcelpociotCollectionviewCollectionViewProxy.h"
 #import "CHTCollectionViewWaterfallLayout.h"
-#import "M13ContextMenu.h"
```

Comment:
and for android, that lib is supported, no? Probably is better go with just the module, I just not want anyone angry on me